cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -490,7 +504,7 @@ class SharedStorageSchema(BaseSchema):
     name = fields.Str(required=True, metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
     storage_type = fields.Str(
         required=True,
-        validate=validate.OneOf(["Ebs", FSX_LUSTRE, FSX_OPENZFS, FSX_ONTAP, "Efs"]),
+        validate=validate.OneOf(["Ebs", FSX_LUSTRE, FSX_OPENZFS, FSX_ONTAP, "Efs", FSX_FILE_CACHE]),
```

Comment:
*Observation to address in a separate PR, but worth starting the discussion for it.*

Why "Efs" is the only one without a corresponding constant. We should fix this.